### PR TITLE
Backend Change for "Experiments with Activity Range" feature on Node Page

### PR DESCRIPTION
**Pull request by dongbohu**
This PR includes changes in the following files:

* `api.py`: added a new **node** filter in **ExperimentResource**;
* `tests.py`: created a test case for the filter, also removed a few occurrences of trailing backslashes;
* `models.py`:  corrected a typo in **Activity** model.

### CODE REVIEW

**Comment by dongbohu** [comment]
I tweaked the filter's implementation a few times and the current one is much shorted than it was.

File: adage/analyze/models.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ class Activity(models.Model):
     value = models.FloatField()
 
     def __unicode__(self):
-        return "Sample %s at Node %s with value %f" % (self.sample.sample,
+        return "Sample %s at Node %s with value %f" % (self.sample.name,
```
Comment (mhuyck) [inline adage/analyze/models.py]: Nice catch. The old way with `.sample` should work, but give the result from calling `__unicode__`, I believe.

File: adage/analyze/tests.py [hunk 2]
```diff
@@ -381,18 +381,18 @@ def setUp(self):
...
         self.create_activities(self.node_counter)
 
-        self.activityURI = self.baseURI + "activity/" + \
-            str(self.random_object(Activity).id) + "/"
-        self.activity_sample_URI = self.baseURI + "activity/?sample=" + \
-            str(self.random_object(Sample).id)
+        self.activityURI = (self.baseURI + "activity/" +
+                            str(self.random_object(Activity).id) + "/")
+        self.activity_sample_URI = (self.baseURI + "activity/?sample=" +
+                                    str(self.random_object(Sample).id))
```
Comment (mhuyck) [inline adage/analyze/tests.py]: More `\` elimination. Huzzah! 🎉

**Comment by dongbohu** [comment]
Thank you Matt. I will take care of the issues you pointed out.

**Comment by dongbohu** [comment]
I also removed `format=json` from the other test cases. It makes the code cleaner. Thank you @mhuyck!

**Comment by mhuyck** [comment]
LGTM. Thanks for noticing the other `format=json` lines... I thought we had already dealt with that so I didn't even bother to look!

**Pull request merged**